src/commands/init.js

```diff
@@ -43,4 +60,13 @@ InitCommand.args = [
     },
 ];
 
+InitCommand.flags = {
+    wrap: flagsHelper.string({
+        char: 'w',
```

Comment:
this means its an alias as in `apify init -w`? if so, i'd rather not do that and keep only the full `--scrapy` flag

if we ever start doing this for more project types, we can add `--wrap` with a `-w` alias, but this feels confusing

Reply:
I decided to try and go with `--wrap=scrapy`, which is basically what you just described.

Calling `apify init -w` would fail with `value required (one of: scrapy)`, or something similar. 
I felt like having it like this is a bit more future proof if we start adding more wrappers. What do you think?